lib/hexpm_web/templates/blog/008-hex-v0.20-released.html.md

```diff
@@ -0,0 +1,48 @@
+## Hex v0.20 released
+
+<div class="subtitle">6 June, 2019 · by Eric Meadows-Jönsson</div>
+
+### Organization managed public packages
+
+A bit more than year ago we released private packages with organizations. With it you can add members to your organization and publish private packages only accessible by the members of the organization. The private packages are published to a separate repository (hosted by hex.pm) making sure they are separate from public packages.
+
+Now we are adding the ability to also manage public packages through your organizations. This means that an open source organization like [Phoenix Framework](/orgs/phoenixframework) or a company such as [Plataformatec](/orgs/plataformatec) can use Hex organizations to manage open source Hex packages just like private packages can be managed by organizations.
+
+When you first publish a new package you will be asked if you want to own it yourself or if it should be managed by an organization you are member of. An already published package can be transferred to an organization by running `mix hex.owner transfer PACKAGE ORGANIZATION`.
+
+Note that public packages managed by an organization does not change its namespace, all public packages still belong to the same global namespace, this feature is only changing the ownership of the package.
+
+### Diffing package versions
+
+With `mix hex.package diff` you can view the difference of two package versions from the command line. By calling `mix hex.package diff decimal 1.6.0..1.7.0` you will see a diff of the two versions. See the video below for examples on how you can combine it with `colordiff` and `less`.
+
+<script id="asciicast-249929" src="https://asciinema.org/a/249929.js" async data-rows=20></script>
+
+For more information and documentation run `mix help hex.package`.
```

Comment:
I think it'd be good to mention that the primary purpose of diff is to see what you're downloading. However, I'm a bit torn as not to scare users, e.g. they don't necessarily need to check every single dependency for changes because we have things like tarball checksums, package publishers etc. (even though I personally think people should check stuff before downloading.)

Maybe it's ok to allude to security aspect like this, with a sentence below?

> The diff command is customisable which allows you to build tools on top of that, e.g.: to scan changed files for security vulnerabilities etc.